cli/src/main/scala/com/codacy/analysis/cli/Main.scala

```diff
@@ -15,11 +16,18 @@ class MainImpl(analyseCommand: Analyze => AnalyseCommand = AnalyseCommand(_, sys
   }
 
   def runCommand(command: CommandOptions): ExitStatus.ExitCode = {
+
+    printCliVersion()
```

Comment:
I am not sure if I love it being print on every command, I would expect a --version or something instead? Or only showing it during the verbose?